task1/src/Counter.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
implicit importing of react can be omitted since 17 version.

Reply:
import { useEffect } from "react";

Reply:
It looks like if we create app using CRA, we should add 'import React'...